src/libsync/propagateuploadv1.cpp

```diff
@@ -175,12 +176,6 @@ void PropagateUploadFileV1::slotPutFinished()
 
     slotJobDestroyed(job); // remove it from the _jobs list
 
-    qDebug() << Q_FUNC_INFO << job->reply()->request().url() << "FINISHED WITH STATUS"
```

Comment:
This is removed here but not in other jobs? Maybe keep it there then too? Even if this is v1..

```
src/libsync/networkjobs.cpp:    qCInfo(lcEtagJob) << "Request Etag of" << reply()->request().url() << "FINISHED WITH STATUS"
src/libsync/networkjobs.cpp:    qCInfo(lcMkColJob) << "MKCOL of" << reply()->request().url() << "FINISHED WITH STATUS"
src/libsync/networkjobs.cpp:    qCInfo(lcLsColJob) << "LSCOL of" << reply()->request().url() << "FINISHED WITH STATUS"
src/libsync/networkjobs.cpp:    qCInfo(lcPropfindJob) << "PROPFIND of" << reply()->request().url() << "FINISHED WITH STATUS"
src/libsync/networkjobs.cpp:    qCInfo(lcProppatchJob) << "PROPPATCH of" << reply()->request().url() << "FINISHED WITH STATUS"
src/libsync/networkjobs.cpp:    qCInfo(lcJsonApiJob) << "JsonApiJob of" << reply()->request().url() << "FINISHED WITH STATUS"
src/libsync/propagatedownload.cpp:            qCInfo(lcGetJob) << "GET of" << reply()->request().url().toString() << "FINISHED WITH STATUS"
src/libsync/propagateremotedelete.cpp:    qCInfo(lcDeleteJob) << "DELETE of" << reply()->request().url() << "FINISHED WITH STATUS"
src/libsync/propagateremotemove.cpp:    qCInfo(lcMoveJob) << "MOVE of" << reply()->request().url() << "FINISHED WITH STATUS"
src/libsync/propagateupload.h:        qCInfo(lcPutJob) << "PUT of" << reply()->request().url().toString() << "FINISHED WITH STATUS"
```